libraries/tools/kotlin-stdlib-gen/src/templates/Ranges.kt

```diff
@@ -76,6 +76,7 @@ object RangeOps : TemplateGroupBase() {
             return TProgression.fromClosedRange(first, last, if (this.step > 0) step else -step)
             """
         }
+        sample("samples.ranges.Ranges.step${primitive!!.stepType}")
```

Comment:
When you modified the generator code, please always run the task generating sources from this code.
See the https://github.com/JetBrains/kotlin/blob/master/libraries/tools/kotlin-stdlib-gen/ReadMe.md for details.

If you run code generation, you'll notice there are changes, please commit them.
Namely, two additional functions get sample reference - `step` for UInt and ULong progressions. 
Makes sense to add the corresponding samples for them.

Reply:
added samples `stepUInt` and `stepULong` . TY for the explanation of generation step.